Add Jaeger to visualize data in Promscale (#961)
timescale/docs: promscale/visualize-data/index.md

# Visualize data in Promscale

> Promscale can be used to visualize the data in tools like

Reviewer: I think this sentence is written backwards. We should say that different tools can be used to visualize data in Promscale and not the other way around.

Reviewer 2: ```suggestion
You can visualize data in Promscale using tools like
```

Is that better?

[Grafana][grafana-homepage] and [Jaeger][jaeger-homepage]

*   **Grafana**: [use Grafana to visualize metrics and traces][grafana-promscale].

> *   **Jaeger**: [use Jaeger Query to visualize traces][jaeger-promscale].

Reviewer: I would not mention Jaeger Query here but just Jaeger. We can mention Jaeger Query in the details of the documentation.
